authorization_services/topics/service-authorization-obtaining-permission.adoc

```diff
@@ -0,0 +1,115 @@
+[[_service_obtaining_permissions]]
+= Obtaining Permissions
+
+To obtain permissions from {project_name} you send an authorization request to the token endpoint. As a result, {project_name} will
+evaluate all policies associated with the resource(s) and scope(s) being requested and issue a RPT with all permissions
+granted by the server.
+
+Clients are allowed to send authorization requests to the token endpoint using the following parameters:
+
+* *grant_type*
++
+This parameter is *required*. Must be `urn:ietf:params:oauth:grant-type:uma-ticket`.
++
+* **ticket**
++
+This parameter is *optional*. The most recent permission ticket received by the client as part of the UMA authorization process.
++
+* **claim_token**
++
+This parameter is *optional*. A string representing additional claims that should be considered by the server when evaluating
+permissions for the resource(s) and scope(s) being requested. This parameter allow clients to push claims to {project_name}. For more details about all supported token formats see.
```

Comment:
Missing the link here: "For more details about all supported token formats see."